config/ui-extensions/apigateways/list

```diff
@@ -0,0 +1,6 @@
+- name: Ready
+  source: status.state
+  widget: Badge
+  highlights:
+    positive:
+      - 'Ready'
```

Comment:
please add cases for error and warning, see:
https://github.com/kyma-project/istio/blob/main/config/ui-extensions/istios/list